src/R/Components/Impl/PackageManager/Implementation/View/PackageList.xaml

```diff
@@ -84,19 +84,18 @@
 
                         <!-- title & summary-->
                         <StackPanel Grid.Column="2">
-                            <TextBlock FontWeight="Bold" TextTrimming="CharacterEllipsis" TextWrapping="NoWrap" VerticalAlignment="Center" Text="{Binding Path=Name, Mode=OneWay}"
-                                   FontSize="{Binding ElementName=_self, Path=FontSize, Converter={x:Static wpf:Converters.FontScale122}}" Margin="4,0,0,0">
-                                <TextBlock.ToolTip>
-                                    <ToolTip Visibility="{Binding Path=Description, Mode=OneWay, Converter={x:Static wpf:Converters.NullOrEmptyIsCollapsed}}">
-                                        <TextBlock Style="{StaticResource TooltipStyle}">
+                            <StackPanel.ToolTip>
+                                <ToolTip Visibility="{Binding Path=Description, Mode=OneWay, Converter={x:Static wpf:Converters.NullOrEmptyIsCollapsed}}">
+                                    <TextBlock Style="{StaticResource TooltipStyle}">
                                         <Run Text="{Binding Path=Name, Mode=OneWay}" FontWeight="Bold" />
                                         <LineBreak />
                                         <Run Text="{Binding Path=Description, Mode=OneWay}" />
```

Comment:
While you at it... I've been getting a lot of binding errors in the Output window. Looks like _self binding doesn't work and that is also why font size is not as large as in NuGet. I managed with
```
FontSize="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType=ListBox}, Path=FontSize, Converter
```
And similarly, there is another case of the same nature in the data import dialog.